tests/integration/mesos/features/sync.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Mesos Sync runner
```

Comment:
for now this is sort of abandoned, but in general i don't see the problem with unit tests being unit tests and integration being behavior-driven. ideally i don't want to even touch our python libs in integration tests?